src/Microsoft.ML.Api/SchemaDefinition.cs

```diff
@@ -327,7 +327,8 @@ public static SchemaDefinition Create(Type userType)
                 // This field does not need a column.
                 // REVIEW: maybe validate the channel attribute now, instead 
                 // of later at cursor creation.
-                if (fieldInfo.FieldType == typeof(IChannel))
+                // Const fields not need to be mapped.
```

Comment:
> not need [](start = 31, length = 9)

do not need

Reply:
Also, this comment only applies to the IsLiteral clause. I suggest to decouple the 2 clauses and have a different comment for each.
```
// Const fields not need to be mapped.
if (fieldInfo.IsLiteral)
                    continue;
// Clause to handle the field that may be used to expose the cursor channel. 
// This field does not need a column.
// REVIEW: maybe validate the channel attribute now, instead 
// of later at cursor creation.
if (fieldInfo.FieldType == typeof(IChannel) || fieldInfo.IsLiteral)
                    continue;
```


---
In reply to: [203871821](https://github.com/dotnet/machinelearning/pull/555#discussion_r203871821) [](ancestors = 203871821)